### PR TITLE
Clarify role of PHONE_NUMBER_CONUNTRY environment variable

### PR DESCRIPTION
**Pull request by mayefsky**
##Description

Existing documentation claims that PHONE_NUMBER_COUNTRY defaults to US; in practice when it is not set, uploads of US numbers fail without the country code.

In https://github.com/MoveOnOrg/Spoke/issues/1824 Max suggested that actually making the behavior default to US might cause a problem. I don't understand why, but this at least brings the documentation in line with current behavior and adds the variable to the lambda example .env (it's already in the other ones) which should prevent this gotcha for most new folks setting up Spoke.

Meta note, this is my first PR so please let me know if I'm doing something off-protocol; thanks!

### CODE REVIEW

File: deploy/lambda-env.json [hunk 1]
```diff
@@ -39,5 +39,6 @@
   "ROLLBAR_ACCESS_TOKEN": "set_this_in_production",
   "ROLLBAR_ENDPOINT": "https://api.rollbar.com/api/1/item/",
   "DST_REFERENCE_TIMEZONE": "America/New_York",
-  "TZ": "America/New_York"
+  "TZ": "America/New_York",
+  "PHONE_NUMBER_COUNTRY": "US"
```
Comment (ibrand) [inline deploy/lambda-env.json:43]: Should we be careful about defaulting this to US after what Max said?

**Review by schuyler1d — APPROVED** [review]
I would rather us fix this bug (it should default rather than fail), but I agree until we fix it, we should document it.

File: deploy/lambda-env.json (hunk 1 above)
Comment (mayefsky) [reply deploy/lambda-env.json:43]: I think the concern was about changing the behavior when unset, versus a concern about the template? FWIW the other templates all to set it--this was just updating the lambda template to match.

Comment (maxkc) [reply deploy/lambda-env.json:43]: I'll write a doc outlining the handful of changes you'd want to make for supporting non-US numbers soon.  [Few ideas are here](https://github.com/MoveOnOrg/Spoke/issues/1754#issuecomment-803264867) if you think of others @mayefsky!

**Pull request merged**